src/main/java/ats/v1/pkb/uses_table/UsesTableImpl.java

```diff
@@ -26,6 +26,12 @@ public void setUses(int statementLineNr, int varIdx) {
             statementList.add(statementLineNr);
     }
 
+    @Override
+    public void setUses(int statementLineNr, List<Integer> varIndices) {
+        for (int varIdx : varIndices)
+            setUses(statementLineNr, varIdx);
```

Comment:
można by dawać lekko inne nazwy metod, bo przez chwilę myślałem że to jakaś dzika rekurencja :D

Reply:
Tu jednak pozwolę sobie skorzystać z function overloading - myśle że jest spoko że korzystając z api mogę wrzucać różne parametry i nie martwić się o nazwę funkcji ;)